iib/workers/tasks/build.py

```diff
@@ -784,6 +886,22 @@ def handle_add_request(
     _update_index_image_build_state(request_id, prebuild_info)
 
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
+        if from_index:
+            msg = 'Checking if bundles are already present in index image'
+            log.info(msg)
+            set_request_state(request_id, 'in_progress', msg)
+            present_bundles = _get_present_bundles(from_index, temp_dir)
+            resolved_bundles = _filter_bundles(present_bundles, resolved_bundles)
```

Comment:
Can we please have a log statement printing out `present_bundles` and `resolved_bundles` so that the user is aware?

Reply:
I added a log which says which bundles will still be added. But I don't think printing every single bundles in the index image is a good idea